public/app/features/scenes/dashboard/DashboardScene.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { PageLayoutType } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { PageToolbar } from '@grafana/ui';
+import { AppChromeUpdate } from 'app/core/components/AppChrome/AppChromeUpdate';
+import { Page } from 'app/core/components/Page/Page';
+
+import { SceneObjectBase } from '../core/SceneObjectBase';
+import { SceneComponentProps, SceneLayout, SceneObject, SceneObjectStatePlain } from '../core/types';
+
+interface DashboardSceneState extends SceneObjectStatePlain {
+  title: string;
+  layout: SceneLayout;
```

Comment:
@ivanortegaalba 

> There is something that looks weird to me. It is having layout as Scene property, but as children for SceneObject. At the declaration time, look inconsistent. Maybe having children for the Scene would solve it.

Is it the above property you mean? We could change that to a children array but it's a bit misleading, we need that to be a single child of type SceneLayout. so an array named children is not ideal.

Reply:
> but as children for SceneObject

You mean SceneGridLayout has a children property while DashboardScene has a layout property?

Reply:
Yeah, this is a good one. Maybe we can call it `root` or `child`. But `layout` is a bit confusing. But this is my personal perception :)